cypress/integration/models/stakeholders.ts

```diff
@@ -31,7 +31,9 @@ export class Stakeholders {
         this.stakeholderEmail = faker.internet.email();
         return this.stakeholderEmail;
     }
-
+    getCurrentStakeholderName(): string {
```

Comment:
There is no need of this function. The stakeholder name, whatever has been set is accessible by the class var - stakeholderName. If you want to use it, then simply create an object of the class and access it.